cloudflare/src/framework/response/apifail.rs

```diff
@@ -1,98 +0,0 @@
-use serde::de::DeserializeOwned;
-use serde_json::value::Value as JValue;
-use std::collections::HashMap;
-use std::error::Error;
-
-use std::fmt;
-use std::fmt::Debug;
-/// Note that APIError's `eq` implementation only compares `code` and `message`.
-/// It does NOT compare the `other` values.
-#[derive(Deserialize, Debug)]
-pub struct ApiError {
-    pub code: u16,
-    pub message: String,
-    #[serde(flatten)]
-    pub other: HashMap<String, JValue>,
```

Comment:
I've never seen this used. Do APIs exist that have error fields other than code and message? Do APIs exist that actually have more than one error??? Either way it doesn't seem like a good way to design this sort of API. If an API has a specific set of data it returns as part of its errors, the Endpoint should design a JSON response around that fact.

Reply:
Unfortunately yes, some APIs do set nonstandard error fields. Last time I checked, the LB team had some rich domain-specific errors here, which is why I included it :(

Reply:
Well luckily with this design we won't need it like this, endpoints can specify their own json error structures.